src/switchcraft/gui/app.py

```
@@ -17,16 +17,20 @@
 from switchcraft.gui.views.intune_view import IntuneView
 from switchcraft.gui.views.settings_view import SettingsView
-from switchcraft.gui.views.ai_view import AIView
+# AIView imported dynamically if needed, or we keep it if it is just a view class (but moved to addon)
+# Actually, AIView is in addon now, so we cannot import it unless we use AddonService or try/except
 from switchcraft.gui.views.analyzer_view import AnalyzerView
-from switchcraft.gui.views.winget_view import WingetView
-from switchcraft.utils.winget import WingetHelper
+from switchcraft.gui.views.winget_view import WingetView # Winget View depends on helper, verify usage
+from switchcraft.gui.views.history_view import HistoryView
+from switchcraft.services.history_service import HistoryService
+from switchcraft.services.history_service import HistoryService
```

Ruff (0.14.8)

Redefinition of unused `HistoryService` from line 31

Remove definition: `HistoryService`

(F811)